ad-joining/register-computer/ad/domain.py

```diff
@@ -281,7 +283,16 @@ def set_computer_zone(self, ou, computer_name, zone):
             raise AlreadyExistsException(e)
 
     def delete_computer(self, computer_dn):
-        self.__connection.delete(computer_dn)
+        try:
```

Comment:
This changes the behavior of the method as before the exception would be passed to the caller. Do we need to change anything upstream?

Reply:
`delete_computer` is only called from one other method (`__cleanup_computers`), and that uses a generic `except Exception ` handler. So the change is safe.